README.md

```diff
@@ -97,178 +91,148 @@
   <p>If you encounter this error, you’ve likely run out of time on your Ignition gateway’s demo period. To resolve, you’ll need to open your Ignition gateway’s webpage and click <code>Reset</code> on the Trial Mode bar. If you’re not signed in to the Ignition gateway webpage, you may need to click <code>Sign In to Reset</code> instead of <code>Reset</code>.</p>
   <img src="images/cannotlogintrialexpired.png" width="273" />
   <br />
-
   <!-- TROUBLESHOOTING/TAG NOT FOUND -->
   <h3 id="error-writing-to-tag-bad_notfound"><span style="font-variant: small-caps;">Error writing to tag, Bad_NotFound</span></h3>
   <p>If you encounter this error, there’s a problem accessing your Ewon’s tags. To resolve, you’ll need to verify that you’ve entered the correct Ewon name when entering the demo.</p>
   <p>If you’ve entered the correct name, but still receive this error, there may be a problem with the Ewon Connector or its configuration. For information on resolving errors with the Ewon Connector, visit the Ignition Ewon Connector GitHub readme at <a href="https://github.com/hms-networks/IgnitionEwonConnector/blob/Ignition-8-master/readme.md" class="uri">https://github.com/hms-networks/IgnitionEwonConnector/blob/Ignition-8-master/readme.md</a> and choose the troubleshooting section.</p>
   <img src="images/tagbadnotfound.png" width="293" />
   <br />
-
   <!-- APPENDICES -->
   <h2 id="appendices"><span style="font-variant: small-caps;">Appendices</span></h2>
-
   <!-- APPENDICES/EWON SETUP -->
   <h3 id="setting-up-an-ewon"><span style="font-variant: small-caps;">Setting Up an Ewon</span></h3>
   <p>Before installation and configuration of the Ewon Ignition Demo, the Ewon must be set up and configured to connect to Talk2M. The <code>System</code>, <code>Internet</code>, and <code>VPN</code> wizards are required to be completed to allow this functionality.</p>
   <p>For more information, and to obtain detailed instructions, please refer to section 6 of the Ewon device’s installation guide.</p>
   <p><strong>Flexy 205 Installation Guide:</strong> <a href="https://websupport.ewon.biz/sites/default/files/ig-0028-00-en-installation-guide-for-flexy-205.pdf#page=22"><em>https://websupport.ewon.biz/sites/default/files/ig-0028-00-en-installation-guide-for-flexy-205.pdf#page=22</em></a><br />
-  <strong>Flexy Base Unit Installation Guide:</strong> <em><a href="https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28" class="uri">https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28</a></em></p>
+    <strong>Flexy Base Unit Installation Guide:</strong> <em><a href="https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28" class="uri">https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28</a></em>
+  </p>
   <br />
```

Comment:
formatting to move the p tag down to its own line also doesn't really fit with the current commit message